src/components/common/Badge/Badge.tsx

```diff
@@ -0,0 +1,33 @@
+import { StyledBadge } from './Badge.style';
+import { PropsBadge } from './Badge.type';
+
+/**
+ * @summary 사용법 <Badge
+      $size={'m'}
+      $state="basic"
+      $isHover={true}
+      $text={'백엔드'}
+    />
+ * @description 공통 Badge 컴포넌트
+ * @param $size font-size의 크기
+ * @param $state basic, focus, hashTag가 있으면 background-color와 color의 색깔을 지정합니다.
+ * @param $hover true false에 따라 hover가 적용됩니다.
+ * @param $text 뱃지안에 들어간 text값 입니다.
+ * @returns
+ */
+
+const Badge = ({ $state, $isHover, $size, $text, ...props }: PropsBadge) => {
+  return (
+    <StyledBadge
+      $size={$size}
+      $state={$state}
+      $isHover={$isHover}
+      $text={$text}
+      {...props}
+    >
+      {$text}
```

Comment:
P3; 만약에 $state가 hashTag인 경우 여기서 #을 붙여주면 좋을 것 같아요! 
```suggestion
      {$state === 'hashTag' ? `#${$text}` : $text}
```

이런식으로...?

Reply:
좋습니다 ! 깜빡했었는데 반영하겠습니다!